src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -2,6 +2,7 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.filter.InvertFilter;
```

Comment:
This is an accidental include I assume?  We definitely don't want to be using this here.